tests/unit/Domain/ValueObject/TypeOfServiceCollectionTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types = 1);
+
+/**
+ * Copyright 2025 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Tests\Unit\Domain\ValueObject;
+
+use PHPUnit\Framework\TestCase;
+use Surfnet\ServiceProviderDashboard\Domain\ValueObject\TypeOfService;
+use Surfnet\ServiceProviderDashboard\Domain\ValueObject\TypeOfServiceCollection;
+
+class TypeOfServiceCollectionTest extends TestCase
+{
+
+    public function testFiltersHidden()
```

Comment:
Personally I would widen the test coverage of the filter. 

1. What happens when there are no items in the collection and you call filter? 
2. Or what happens if all values are filtered out?

I know these are no cases that should ever occur. But it might be good to test the collection according to those possibilities.

Reply:
Thanks for adding the test coverage :1st_place_medal: